src/main/resources/assets/computercraft/lua/rom/help/settings.txt

```diff
@@ -11,6 +11,7 @@ Default Settings:
 shell.autocomplete - enables auto-completion in the Shell.
 lua.autocomplete - enables auto-completion in the Lua program.
 edit.autocomplete - enables auto-completion in the Edit program.
+edit.extension - edit will add .lua extension
```

Comment:
Probably should read `edit will add .lua when no extension is specified` or something.